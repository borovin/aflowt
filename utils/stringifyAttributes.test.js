const test = require('ava')
const stringifyAttributes = require('./stringifyAttributes')

test('stringifyAttributes test', t => {
  const snapshot = stringifyAttributes({
    'a': 1,
    'b': 'b',
    'c': false,
    'd': true,
    'e': {a: 1}
  })

  t.snapshot(snapshot)
})
