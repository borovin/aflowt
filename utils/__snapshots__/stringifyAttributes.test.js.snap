// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`stringifyAttributes test 1`] = `"a=\\"1\\" b=\\"b\\"  d e=\\"{\\"a\\":1}\\""`;
